src/utils/intl.ts

```diff
@@ -14,7 +19,14 @@ const cache = createIntlCache();
 export const intl = (messages: Record<string, string> = {}, locale = 'en') =>
   createIntl(
     {
-      messages,
+      messages: {
+        ...colonyMessages,
```

Comment:
So that the intl object created outside a react component has access to all the same messages as one created inside a react component